ecl-patch/0001-Enable-utilisation-of-commercial-licenses.patch

```diff
@@ -50,44 +50,20 @@ index 18afa57..6be19dc 100644
 -//      libraryDependencies += Cinnamon.library.cinnamonPrometheusHttpServer,
 -//      libraryDependencies += Cinnamon.library.cinnamonAkkaHttp,
 -//      libraryDependencies += Cinnamon.library.cinnamonOpenTracingZipkin,
--//      libraryDependencies += Cinnamon.library.cinnamonCHMetricsElasticsearchReporter,
--//      AssemblyKeys.assembly := Def.task {
--//        JavaAgentKeys.resolvedJavaAgents.value.filter(_.agent.name == "Cinnamon").foreach { agent =>
--//          sbt.IO.copyFile(agent.artifact, target.value / "cinnamon-agent.jar")
--//        }
--//        AssemblyKeys.assembly.value
--//      }.value,
--//      assemblyMergeStrategy in assembly := {
--//        case "cinnamon-reference.conf" => MergeStrategy.concat
--//        case x =>
--//          val oldStrategy = (assemblyMergeStrategy in assembly).value
--//          oldStrategy(x)
--//      }
+-//      libraryDependencies += Cinnamon.library.cinnamonCHMetricsElasticsearchReporter
 -//    )
 +    .settings(
 +      libraryDependencies += Cinnamon.library.cinnamonPrometheus,
 +      libraryDependencies += Cinnamon.library.cinnamonPrometheusHttpServer,
 +      libraryDependencies += Cinnamon.library.cinnamonAkkaHttp,
 +      libraryDependencies += Cinnamon.library.cinnamonOpenTracingZipkin,
-+      libraryDependencies += Cinnamon.library.cinnamonCHMetricsElasticsearchReporter,
-+      AssemblyKeys.assembly := Def.task {
-+        JavaAgentKeys.resolvedJavaAgents.value.filter(_.agent.name == "Cinnamon").foreach { agent =>
-+          sbt.IO.copyFile(agent.artifact, target.value / "cinnamon-agent.jar")
-+        }
-+        AssemblyKeys.assembly.value
-+      }.value,
-+      assemblyMergeStrategy in assembly := {
-+        case "cinnamon-reference.conf" => MergeStrategy.concat
-+        case x =>
-+          val oldStrategy = (assemblyMergeStrategy in assembly).value
-+          oldStrategy(x)
-+      }
++      libraryDependencies += Cinnamon.library.cinnamonCHMetricsElasticsearchReporter
 +    )
        .enablePlugins(DockerPlugin, JavaAppPackaging)
        .settings(
-       mappings in Universal += file("librpi_ws281x.so") -> "lib/librpi_ws281x.so",
+         mappings in Universal ++=
```

Comment:
librpi disappeared?

Reply:
Seems weird... I pushed a commit in which I changed the name of the patch file. Also, I tested the previous version which didn't seem to delete that `librpi_ws281x.so` line...
Just to be sure, I pulled the PR and ran the `ecl` script and verified that it works as expected.